lib/modern_searchlogic/column_conditions.rb

```diff
@@ -72,6 +72,15 @@ def searchlogic_arel_alias(searchlogic_suffix, arel_method, options = {})
         end
       end
 
+      def searchlogic_active_record_alias(searchlogic_suffix, options = {}, &block)
+        searchlogic_suffix_condition "_#{searchlogic_suffix}" do |column_name, *args|
+          raise ArgumentError, "wrong number of arguments (0 for >= 1)" if args.empty?
```

Comment:
I think you can update the declarations that use `searchlogic_active_record_alias` to pass in `:any_or_all => true` and then remove this check.

Reply:
This will still require adding a check for `any_or_all` in the method and raising an error